fuse.py

```diff
@@ -376,18 +377,27 @@ class fuse_operations(Structure):
         ('flag_reserved', c_uint, 29),
     ]
 
+# When importing "fuse", you can check this with "hasattr(fuse, 'NANOSECOND_INT_AVAILABLE')"
+NANOSECOND_INT_AVAILABLE = True
 
-def time_of_timespec(ts):
-    return ts.tv_sec + ts.tv_nsec / 10 ** 9
+def time_of_timespec(ts, nanosecond_int = False): # nanosecond_int is opt-in by default
```

Comment:
The comment `# nanosecond_int is opt-in by default` is already made explicit by `nanosecond_int` defaulting to `False`.

Also, as per [PEP8](https://www.python.org/dev/peps/pep-0008/#other-recommendations) guidelines, I suggest removing the spaces surrounding the `=` operator. So `nanosecond_int = False` should become `nanosecond_int=False`